ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5655,4 +5655,120 @@ RFVSC2X:
+  - return_water_isolation_valve_status
+  - heating_supply_water_temperature_setpoint
+  - chilled_supply_water_temperature_setpoint
+  uses:
+  - heating_supply_water_temperature_sensor
+  - heating_return_water_temperature_sensor
+  - chilled_supply_water_temperature_sensor
+  - chilled_return_water_temperature_sensor
+  - cooling_output_percentage_command
+  - heating_output_percentage_command
+
+HHCD42:
+  description: "Hydronic heating and cooling distribution monitoring for change over group with a heat and a cold exchanger. Primary side: Four-pipe installation 
+                with flow control valves for cooling and heating. Two pipe installation with isolation valves on the secondary side towards the radiant panels. 
+                Heating or cooling only"
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - heating_supply_water_isolation_valve_status
+  - heating_return_water_isolation_valve_status
+  - chilled_supply_water_isolation_valve_status
+  - chilled_return_water_isolation_valve_status
+  - chilled_water_valve_percentage_command
+  - heating_water_valve_percentage_command
+  - secondary_heating_supply_water_temperature_sensor
+  - secondary_heating_return_water_temperature_sensor
+  - secondary_chilled_supply_water_temperature_sensor
+  - secondary_chilled_return_water_temperature_sensor
+  - heating_request_count
+  - cooling_request_count
+  opt_uses:
+  - average_zone_air_temperature_sensor
+  - primary_chilled_supply_water_temperature_sensor
+  - primary_heating_supply_water_temperature_sensor
+
+HHCD44:
+  description: "Hydronic heating and cooling distribution monitoring for change over group with a heat and a cold exchanger. Primary side: Four-pipe installation 
+                with flow control valves for cooling and heating. Four pipe installation with isolation valves on the secondary side towards the radiant panels.
+                Heating and cooling in parallel"
+  is_abstract: true
+  implements:
+  - MONITORING
+  uses:
+  - heating_supply_water_isolation_valve_status
+  - heating_return_water_isolation_valve_status
+  - chilled_supply_water_isolation_valve_status
+  - chilled_return_water_isolation_valve_status
+  - chilled_water_valve_percentage_command
+  - heating_water_valve_percentage_command
+  - secondary_heating_supply_water_temperature_sensor
+  - secondary_heating_return_water_temperature_sensor
+  - secondary_chilled_supply_water_temperature_sensor
+  - secondary_chilled_return_water_temperature_sensor
+  - heating_request_count
+  - cooling_request_count
+  opt_uses:
+  - average_zone_air_temperature_sensor
+  - primary_chilled_supply_water_temperature_sensor
+  - primary_heating_supply_water_temperature_sensor
+
+DFVSCS:
+  description: "Variable speed control for discharge fans (simple)."
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - discharge_fan_run_status
+  uses:
+  - discharge_fan_speed_percentage_sensor
+  - discharge_fan_run_command
+
+DXSZC:
```

Comment:
Use DXZC, and if you dont have the compressor run command then define it as missing (I assume its not available because it commands the compressor in the onboard controller and cannot be remotely enabled).

Reply:
I'll remove it as well